### PR TITLE
Updated UG documentation  for Free text Annotation, Stamp, Eraser and toolbar.

### PR DESCRIPTION
**Pull request by KSilambharasanSF4548**
Description :
Tasks  : 
1.Add and modify free text annotations using built-in toolbar
2.Add and modify stamp annotations using built-in toolbar
Links : 
1.https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/890415/
2.https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_sprints/backlog/PdfViewerMAUI/Mobile%20and%20Desktop/2024Jun1_PDFViewer_MAUI?workitem=890414

### CODE REVIEW

File: MAUI/PDF-Viewer/Free-Text.md [hunk 1]
```diff
@@ -184,11 +196,11 @@ Private void PdfViewer_FreeTextModalViewDisappearing(object? Sender, EventArgs e
 {% endhighlight %} 
 {% endtabs %}
 
-### Supressing the free text modal view and implement your own UI
+### Suppressing the free text modal view and implement your own UI
 
-The [Sfpdfviewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to supress the free text modal view and use your own UI in its place. This can be achieved by setting the `AnnotationModalViewAppearingEventArgs.Cancel` property to `true` in the `FreeTextModalViewAppearing` event handler. 
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to suppress the free text modal view and use your own UI in its place. This can be achieved by setting the `AnnotationModalViewAppearingEventArgs.Cancel` property to `true` in the `FreeTextModalViewAppearing` event handler.
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Free-Text.md:201]: Since the APIs are in live now, can this be API reference linked?

File: MAUI/PDF-Viewer/Free-Text.md [hunk 2]
```diff
@@ -217,4 +229,4 @@ Private void customDialogOkButton_Clicked(object sender, EventArgs e)
 {% endhighlight %} 
 {% endtabs %}
 
-N> For WinUI and MacCatalyst platforms, there is no separate modal view to receive text input from the users. As a result, the `FreeTextModalViewAppearing` and `FreeTextModalViewDisappearing` events are not applicable for these platforms. 
\ No newline at end of file
+N> For WinUI and MacCatalyst platforms, there is no separate modal view to receive text input from the users. As a result, the `FreeTextModalViewAppearing` and `FreeTextModalViewDisappearing` events are not applicable for these platforms.
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Free-Text.md:232]: There is unwanted black border appears. can you please remove this?

image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: MAUI/PDF-Viewer/Free-Text.md (hunk 1 above)
Comment (VikramDevaraj31) [reply MAUI/PDF-Viewer/Free-Text.md]: Feedback addressed

File: MAUI/PDF-Viewer/Free-Text.md (hunk 2 above)
Comment (VikramDevaraj31) [reply MAUI/PDF-Viewer/Free-Text.md]: Feedback addressed

**Review by Deepak1799 — APPROVED** [review]
Approved

**Pull request merged**